code/modules/antagonists/changeling/powers/headcrab.dm

```diff
@@ -37,13 +37,15 @@
 		user.investigate_log("has been gibbed by headslug burst.", INVESTIGATE_DEATHS)
 	user.gib()
 	. = TRUE
-	addtimer(CALLBACK(src, PROC_REF(spawn_headcrab), stored_mind, user_turf, organs), 3 SECONDS)
+	// So that the headslug does not get blown up, this is ugly
+	addtimer(CALLBACK(src, PROC_REF(spawn_headcrab), stored_mind, user_turf, organs), 1 SECONDS)
 
 /datum/action/changeling/headcrab/proc/spawn_headcrab(datum/mind/stored_mind, turf/spawn_location, list/organs)
 	var/mob/living/basic/headslug/crab = new(spawn_location)
 	for(var/obj/item/organ/I in organs)
 		I.forceMove(crab)
 
 	stored_mind.transfer_to(crab)
+	crab.key = stored_mind.key
```

Comment:
the above line (`stored_mind.transfer_to(crab)`) is supposed to handle moving the key over, do you know why it doesn't?

Reply:
I changed it to stored_mind.transfer_to(crab, force_key_move = TRUE) and that worked, which implies that the changelings mind.active is being set to FALSE somewhere. I couldn't really figure out why or where. If this isn't a good solution I can do more digging though.

Reply:
![image](https://github.com/tgstation/tgstation/assets/34697715/b16638a4-5bbf-4aab-8fe3-be97c6eabc2d)

I believe this is a consequence of the underlying framework, as the `active` thing doesn't get set to true? This was the key change in headslug code that may have spawned this back in #75601, I removed the line because w*llard told me to remove the `origin` variable that they used to have and I didn't think it was consequential (which may be why this was working in my first implementations, I hadn't tested it after reviews. apologies).

Reply:
i think having `force_key_move = TRUE` is fine to keep regardless of this since it's very explicit in intent of the proc (auto-documenting code) but this is likely why you don't see active be set to `TRUE` anywhere